CONTRIBUTING.md

```diff
@@ -24,7 +28,7 @@ support@pointonenav.com.
 
 ## Submitting Changes
 
-### Creating A Branch
+### Create A Branch
```

Comment:
You changed this from gerund to imperative, but all other section titles, including those in the table of contents, are gerunds.